docs/userguide/AutoComplete.adoc

```diff
@@ -21,15 +21,18 @@ If your "abc" is your input in your commandbox, only words that starts with "abc
 - When prefixes are suggested you should select all prefixes that you need then type in respective sub-details +
 Eg: Select all prefixes "pat-add n/ p/ e/ a/ t/" before adding sub-details "pat-add n/John p/98762615 e/john@example.com a/exampleaddress t/exampletag vt/examplevisittodo"
 - If no words appear in the autocomplete panel, this can only mean 2 thing. +
-1) Your input is finished, there is no more words to be suggested +
+1) Your full command is finished, there is no more words to be suggested +
 2) Your input is wrong and does not match any of the words to be suggested. (Try to change your input by backspacing)
 
 === Selecting a suggested words
-*UP*, *DOWN* key +
-You can use the up and down key in your keyboard to move from one selected word to another.
+`*UP*`, `*DOWN*` key +
+You can use the `*UP*` and `*DOWN*` key in your keyboard to move from one selected word to another. Description of the suggested words will be shown as you select them to guide you in choosing.
 
-*RIGHT* key +
-Confirm your selection by pressing the right key in your keyboard. This will auto-fill the selected word to your current words in the command box text field. Doing this will also perform a auto correction of your input in order to fit the correct sequence of commands. +
+`*RIGHT*` key +
+Confirm your selection by pressing the `*RIGHT*` key in your keyboard. This will auto-fill the selected word to your current words in the command box text field. Doing this will also perform a auto correction of your input in order to fit the correct sequence of commands. +
```

Comment:
```suggestion
Confirm your selection by pressing the `*RIGHT*` key in your keyboard. This will auto-fill the selected word to your current words in the command box text field. Doing this will also perform an auto correction of your input in order to fit the correct sequence of commands. +
```